### DISCUSSION TITLE
Possible with CDN?

### DISCUSSION THREAD

**Discussion by thdoan** [FIameCaster/prism-code-editor#10] [Q&A] [answered]
Hi, if I don't have access to a build system, is it possible to use this completely from a CDN?

**Comment by jonpyt** [comment, marked as answer]
Yes, the package is ESM, so it will run in a browser without a build step. If you want to use unpkg, #3 explains how. Obviously replace `@2.2.2` in the links with `@3.3.1` for the current version. *Note:* prism grammar imports where changed in version 3, so the example in #3 must be changed slightly.

The code in the package is not minified, so using a CDN here is heavily discouraged in production, but for prototyping this is not an issue.

**Reply by thdoan** [reply]
Thanks, this helped. I can see how it gets tricky really fast if you want to download and use these files locally because of all the interconnected imports 😅
